custom_reports/http_api/definitions/real_time_query_request.yaml

```diff
@@ -0,0 +1,253 @@
+type: object
+properties:
+  website_id:
+    type: string
+    format: uuid
+    description: ID of a website or a meta site.
+    example: 70236925-724f-404e-b634-7b5633bc0bcc
+  columns:
+    type: array
+    description: Column definitions for the query.
+    items:
+      type: object
+      properties:
+        column_id:
+          type: string
+          description: ID of the column (either a dimension or a metric).
+        transformation_id:
+          type: string
+          description: |
+            Optional ID of the transformation. Must be supported by the
+            column that was specified.
+        goal_id:
+          type: integer
+          description: |
+            Optional ID of a goal. Available only for metrics related to goals.
+      required:
+        - column_id
+    example: [{"column_id":"device_type"},{"column_id":"revenue","transformation_id":"sum"}]
+  lookback_window:
+    type: int
+    format: date
```

Comment:
Is it correct ? Format date is for string (for example, 2017-07-21) https://swagger.io/docs/specification/data-models/data-types/#format

Reply:
of course not, good catch :+1: